### ISSUE TITLE
allow call AllowTemplate by alias

### ISSUE THREAD

**Issue by Myster** [AffinityID/uMigrate#36]
See extension method implementation of this feature request in the wiki here: https://github.com/AffinityID/uMigrate/wiki/Create-Template

**Comment by ashmind** [comment]
@Myster Makes sense, thanks for the sample -- I'll try to integrate it next time I go through GH issues.
